web-app/src/games/games.service.spec.ts

```diff
@@ -164,4 +177,71 @@ describe('GameService', () => {
 `).board,
     );
   });
+
+  describe('getGameAndStatus', () => {
+    it('should return "INITIALIZED" if there is only one player', () => {
+      // GIVEN
```

Comment:
I think these comments(GIVEN/WHEN/THEN) aren't required anymore because blank lines make it already explicit !

Reply:
I respectfully disagree 😁 
I suggest we summon the advice of a 3rd party to settle this 😁